src/adapters/postgres/cohortMembers-adapter.ts

```
@@ -4504,4 +4548,363 @@ export class PostgresCohortMembersService {
+   *
+   * @param cohortId - The cohort ID to fetch courses for
+   * @returns Promise with array of courses or empty array if none found
+   */
+  private async fetchLMSCoursesForCohort(cohortId: string) {
+    try {
+      const lmsBaseUrl = process.env.LMS_SERVICE_URL;
+      const tenantId = process.env.DEFAULT_TENANT_ID;
+      const organisationId = process.env.DEFAULT_ORGANISATION_ID;
+
+      if (!lmsBaseUrl || !tenantId || !organisationId) {
+        throw new Error('LMS service configuration missing');
```

ESLint

[error] Replace `'LMS·service·configuration·missing'` with `"LMS·service·configuration·missing"`

(prettier/prettier)